docs/docsite/rst/porting_guide_2.4.rst

```diff
@@ -37,6 +37,16 @@ Since there is no longer a single inventory, the 'implicit localhost' doesn't ge
 
 A bug was fixed with the inventory path/directory, which was defaulting to the current working directory. This caused ``group_vars`` and ``host_vars`` to be picked up from the current working directory instead of just adjacent to the playbook or inventory directory when a host list (comma separated host names) was provided as inventory.
 
+Initial playbook relative group_vars and host_vars
+--------------------------------------------------
+
+In Ansible versions prior to 2.4, the inventory system would maintain context of the initial playbook that was executed. This allowed successively included playbooks to inherit playbook file-relative ``group_vars`` and ``host_vars``.
```

Comment:
@dharmabumstead proposed this in channel:
> In Ansible versions prior  to 2.4, the inventory system would maintain context of the initial playbook that was executed. This allowed successively included playbooks to inherit ``group_vars`` and ``host_vars`` placed relative to the playbook file.

jctanner said this sounds like this applies to all playbook-relative vars and suggested this:

> In Ansible versions prior  to 2.4, the inventory system would maintain context of the initial playbook that was executed. This allowed successively included playbooks to inherit ``group_vars`` and ``host_vars`` placed relative to the toplevel playbook file.

Reply:
I agree that the recommendation change sounds better. I realized I didn't put enough emphasis on the fact that this is in relation to included playbooks in subdirectories.